src/components/metadata/index.js

```diff
@@ -52,6 +53,11 @@ const MetaData = ({
     onToggleNodeSelected(null);
   };
 
+  const onExpandPlotClick = () => {
+    // Deselecting a node automatically hides MetaData panel
```

Comment:
Don't think this comment is related to this callback ( think it's copied from `onCloseClick`?) please delete :)